pymc/tests/test_distributions.py

```diff
@@ -2619,6 +2619,21 @@ def test_interpolated_transform(self, transform):
             assert not np.isfinite(m.compile_logp()({"x": -1.0}))
             assert not np.isfinite(m.compile_logp()({"x": 11.0}))
 
+    def test_grw(self):
```

Comment:
Moved this back here, because of the inheritance issues that you found @canyon289. We will eventually refactor this test class and split the tests across different modules, but there's no reason to do it in this PR

Reply:
Great marking for another issue